src/bind/Toolchain.cxx

```diff
@@ -101,3 +107,32 @@ void Toolchain::set_free() {
         return Godot::print("Warning: BoardRunner queued to be freed while still building");
     queue_free();
 }
+
+Array Toolchain::find_compilers() {
+    Array result;
+
+    auto compilerDefault = make_ref<Toolchain::CompilerInformation>();
+    compilerDefault->name = "Default";
+    compilerDefault->path = "-";
+    compilerDefault->version = "-";
+    result.push_back(compilerDefault);
+
+    auto compilers = tc->find_compilers();
+    for (const smce::Toolchain::CompilerInformation ci : compilers) {
+        auto compiler = make_ref<Toolchain::CompilerInformation>();
```

Comment:
make a static method on Toolchain::CompilerInformation

e.g. `Ref<CompilerInformation> from(smce::Toolchain::CompilerInformation&);`

otherwise functionality isnt tied to the class itself